### PR TITLE
SortMeRNA updates and latest tables

### PR DESCRIPTION
**Pull request by wasade**
Bit of a big PR...

### CODE REVIEW

**Comment by wasade** [comment]
@antgonza, possible to do the review on `beta_sample_Rarefaction`? 

Adding GG/PGP/HMP SortMeRNA tables momentarily. Would be great to get this merged.

**Comment by wasade** [comment]
GG/PGP/HMP smr tables were actually in. Didn't have final AG metadata in though. This is good for final review

**Comment by JWDebelius** [comment]
The new metadata still has the field coding errors, as far as I can tell. Are those staying until round 15?

**Comment by wasade** [comment]
I was updating those this morning, commit is coming momentarily

**Comment by JWDebelius** [comment]
The only thing I'm still seeing is SEX.

**Comment by wasade** [comment]
good catch, thanks

**Comment by JWDebelius** [comment]
Looks good to me. :+1:

File: scripts/beta_sample_rarefaction.py [hunk 1]
```diff
@@ -116,10 +124,11 @@ def main():
                     label=label)
 
     if y_max:
-        axis([0, max_x, 0, max_y])
-    else:
         axis([0, max_x, 0, y_max])
-
+    else:
+        axis([0, max_x, 0, max_y])
+    print y_max, max_y
```
Comment (antgonza) [inline scripts/beta_sample_rarefaction.py]: You do not need this print, right?

**Pull request merged**